crates/modules/process-monitor/probes.bpf.c

```diff
@@ -156,20 +157,20 @@ The array size MUST be greater than 72.
 
 ### Success:
     the return value is:
-      - `0`: it's a docker container id
-      - `1`: it's a podman container id
+      - `1`: it's a docker container id
+      - `2`: it's a podman container id
```

Comment:
as we have already discussed it breaks the FFI `enum ContainerEngineKind`. the mappings become invalid